weather_lookup/app.R

```diff
@@ -200,7 +190,14 @@ server <- function(input, output, session) {
               panel.grid.minor = element_line(color = "grey85", size = 0.2))
```

Comment:
Just curious, why is this here?

Reply:
I had thought that the gray lines were less confusing given the temperature line is white itself. However, this was before the dark-mode was added so I'm less certain now

Reply:
Either way a comment should have been there to explain the "why", added one.